backpack-common/src/main/java/net/skyscanner/backpack/calendar2/data/CalendarCell.kt

```diff
@@ -98,40 +100,103 @@ internal fun CalendarCellDay(
     yearMonth: YearMonth,
     selection: CalendarSelection,
     params: CalendarParams,
-): CalendarCell.Day = CalendarCell.Day(
-    date = date,
-    yearMonth = yearMonth,
-    info = params.cellsInfo[date] ?: CellInfo.Default,
-    outOfRange = date !in params.range,
-    contentDescription = date.format(params.dateContentDescriptionFormatter),
-    text = buildSpannedString {
-        val span = TtsSpan.DateBuilder()
-            .setDay(date.dayOfMonth)
-            .setMonth(date.month.ordinal)
-            .setYear(date.year)
-            .setWeekday(date.dayOfWeek.value)
-            .build()
-        append(date.dayOfMonth.toString(), span, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
-    },
-    selection = when (selection) {
-        is CalendarSelection.None -> null
-        is CalendarSelection.Single -> when (date) {
-            selection.date -> CalendarCell.Selection.Single
+): CalendarCell.Day {
```

Comment:
we should have tests for the different states/content descriptions here (i think that is `CalendarInfoTests` but @bvitaliyg should be able to confirm

Reply:
I added some tests in BpkCalendarTests. Do you mean those, or should I add some other type of tests as well?

Reply:
these are a good start, yes - we should be able to test the logic, rather than the ui, though. looking closer it might be in `CalendarCellsLayoutTests` - ideally you'd cover all the different options/hints in tests

Reply:
Thanks Maria! I've created a separate file for them: CalendarAccessibilityLabelTests.